subgraph/package.json

```diff
@@ -22,7 +22,22 @@
     "test": "graph test",
     "deploy:studio": "graph deploy --node https://api.studio.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ unlock-protocol-mainnet",
     "ci": "yarn prepare && yarn codegen && yarn test -l",
-    "lint": "eslint --resolve-plugins-relative-to ../packages/eslint-config --ext .ts,.js src/ bin/"
+    "lint": "eslint --resolve-plugins-relative-to ../packages/eslint-config --ext .ts,.js src/ bin/",
+    "generate-arbitrum": "mustache config/arbitrum.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-avalanche": "mustache config/avalanche.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-base-goerli": "mustache config/baseGoerli.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-base": "mustache config/base.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-bsc": "mustache config/bsc.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-celo": "mustache config/celo.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-gnosis": "mustache config/gnosis.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-goerli": "mustache config/goerli.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-mainnet": "mustache config/mainnet.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-mumbai": "mustache config/mumbai.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-optimism": "mustache config/optimism.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-palm": "mustache config/palm.json subgraph.template.yaml > subgraph.yaml && graph codegen",
+    "generate-polygon": "mustache config/polygon.json subgraph.template.yaml > subgraph.yaml && graph codegen",
```

Comment:
I think we could refactor these generate functions as well, so they just take the network name as param

Reply:
Have made into one script which just requires the json file name `yarn generate-manifest arbitrum.json` as an example

Reply:
@clemsos Is this sufficient?